src/jobs/rss-news.js

```diff
@@ -61,17 +62,30 @@ async function processSchool(school) {
 }
 
 async function run() {
+	logger.info('will update rss feeds...');
+	const errors = [];
 	await database.connect();
 
 	const cursor = schoolModel.find({}).cursor();
 	let school = await cursor.next();
 	while (school) {
-		await processSchool(school);
+		try {
+			await processSchool(school, errors);
+		} catch (error) {
+			const message = `error updating rss news for school '${school._id}', continue...`;
+			errors.push(message);
```

Comment:
The words error as error and errors as stack for error messages is nearly the same word, but is used in different ways. Maybe you can rename error in err, or error in errorMessages, for more human readable code?